Tools/iobench/iobench.py

```diff
@@ -29,9 +29,9 @@ def text_open(fn, mode, encoding=None):
         return open(fn, mode)
 
 def get_file_sizes():
-    for s in ['20 KB', '400 KB', '10 MB']:
+    for s in ['20 KiB', '400 KiB', '10 MiB']:
```

Comment:
This file should be kept compatible with both Python 2.6 and Python >= 3.0.

Changing the output format will make harder comparing results between versions.

Reply:
This script is compatible with Python 2.7 (I didn't test Python 2.6).

If the output must be the same with the script in Python 2.7, I can update Python 2.7 as well.

@pitrou: Since you wrote the script, what do you think?

Reply:
No problem from me.